src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,51 @@
+package duke.command;
+
+import java.util.Locale;
+
+import duke.exception.DukeException;
+import duke.util.Storage;
+
+/**
+ * RecognisedCommand that Duke can execute
+ * @author Nephelite
+ * @version 0.2
+ */
+public abstract class Command {
+    /**
+     * All commands recognised by Duke
+     */
+    public enum RecognisedCommand {
+        BYE, LIST, HELP, MARK, UNMARK, DELETE, TODO, DEADLINE, EVENT, FIND, INVALID;
+    }
+
+    /**
+     * Executes the command
+     * @param storage Duke's storage system for tasks
+     * @return Duke's response to the execution of the command
+     * @throws DukeException for invalid inputs
+     * @since 0.2
+     */
+    public abstract String execute(Storage storage) throws DukeException;
+
+    /**
+     * Checks if the command leads to the end of the current Duke session
```

Comment:
Could you add an empty line between description and parameter section? I saw this in several other areas of the code. Reference: https://se-education.org/guides/conventions/java/intermediate.html - "Empty line between description and parameter section"

Reply:
Oh whoops, I did not notice that was a part of the convention. Thanks!